System/Process/Internals.hs

```diff
@@ -414,6 +429,8 @@ foreign import ccall unsafe "runInteractiveProcess"
         -> Ptr FD
         -> Ptr FD
         -> Ptr FD
+        -> Ptr CGid
```

Comment:
The research I did before making my comment implied that:

1. Signedness is implementation specific, and will vary by platform
2. Even on platforms with an unsigned type, the `-1` value is considered reserved

I could be mistaken on this, however. I'm OK with either approach, it just struck me as strange to use a pointer for integral data.